app/src/test/java/com/bc/gordiansigner/model/DescriptorTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.bc.gordiansigner.model
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class DescriptorTest {
+
+    @Test
+    fun `single sig descriptor test`() {
```

Comment:
I don't have strong opinion for naming the test method. Just want to unify for later. Which one do you prefer?
`single sig descriptor test` or `singleSigDescriptorTest`

Reply:
the test needs to be easy to read, so I think should write as a sentence so that it will be easier.